go/apps/api/routes/chproxy_verifications/handler.go

```diff
@@ -48,7 +48,7 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 			fault.Public("The provided token is invalid."))
 	}
 
-	events, err := zen.BindBody[[]schema.KeyVerificationRequestV1](s)
+	events, err := zen.BindBody[[]schema.KeyVerification](s)
```

Comment:
What do you think about adjusting the v1 endpoints to send over the new fields too? 

Applies to ratelimits too